src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -61,18 +132,51 @@ class XcvrInfoDB(str, Enum):
     "tx4power"    : "TX Power",
 }
 
-QSFP_LANES = (1, 2, 3, 4)
+XCVR_SENSOR_INDEX_MAP = {
+    "temperature" : 1,
+    "tx1power"    : 2,
+    "tx2power"    : 3,
+    "tx3power"    : 4,
+    "tx4power"    : 5,
+    "rx1power"    : 6,
+    "rx2power"    : 7,
+    "rx3power"    : 8,
+    "rx4power"    : 9,
+    "tx1bias"     : 10,
+    "tx2bias"     : 11,
+    "tx3bias"     : 12,
+    "tx4bias"     : 13,
+    "voltage"     : 14,
+}
 
+NOT_AVAILABLE = 'N/A'
+QSFP_LANES = (1, 2, 3, 4)
 
-def get_transceiver_data(xcvr_info):
+def is_null_str(value):
     """
-    :param xcvr_info: transceiver info dict
-    :return: tuple (type, hw_version, mfg_name, model_name) of transceiver;
-    Empty string if field not in xcvr_info
+    Indicate if a string value is null
+    :param value: input string value
+    :return: True is string value is empty or equal to 'N/A' or 'None'
     """
+    if not value or value == NOT_AVAILABLE or value == str(None):
```

Comment:
>str(None) [](start = 55, length = 9)

use `None` ?

Reply:
Not sure I understand.

Reply:
I mean why not use `'None'`  directly?

---
In reply to: [511724801](https://github.com/Azure/sonic-snmpagent/pull/168#discussion_r511724801) [](ancestors = 511724801)